sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -230,6 +238,12 @@ export class InstitutionController extends BaseController {
     } else if (!institutionUser.user.isActive) {
       throw new UnprocessableEntityException("Not an Active User.");
     }
+    // checking if user belong to logged-in users institution
```

Comment:
Now that we have the IsInstitutionAdmin added we can remove the lines 234-240 because if the user is not present or is not active it will be blocked before reach the controller.
Please consider removing the above-mentioned lines.